lib/cdo/activity_constants.rb

```diff
@@ -13,6 +13,7 @@ module ActivityConstants
   FREE_PLAY_RESULT = 30
   BEST_PASS_RESULT = 100
 
+  UNREVIEWED_SUBMISSION_RESULT = 1000
```

Comment:
Do there already exist records with value `1000` in our DB? For some reason, I'm thinking it might have been used previously.